### PR TITLE
Fix Markdown Line Length

### PR DESCRIPTION
**Pull request by john0isaac**
# Description
This PR ignores line length in the markdown to pass the workflow and fixes other markdown issues.

## Checklist
- [x] I have read the contributing guidelines.
- [-] I have written tests for the changes, if applicable.
- [x] I have added necessary documentation.

### Related Issues
Please reference related issues here:

### CODE REVIEW

**Comment by kjaymiller** [comment]
I'm personally not a fan of [MD013](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md) as I think some markdown parsers will mess it up and adding arbitrary `\n` into text in the middle of sentences doesn't seem right. 

I don't mind the other two fixes but I think the actual change should be to add a `markdownlint.jsonc` that ignores MD013.

**Comment by kjaymiller** [comment]
for more information - https://github.com/DavidAnson/markdownlint-cli2?tab=readme-ov-file#markdownlint-cli2jsonc

File: README.md
Comment (kjaymiller) [inline README.md:1]: see other comments

**Review by kjaymiller — CHANGES_REQUESTED** [review]

**Comment by john0isaac** [comment]
done

**Review by kjaymiller — APPROVED** [review]

**Comment by kjaymiller** [comment]
Did we check that the tests pass.. I don't know if I have a linter for it or not.

**Comment by john0isaac** [comment]
Which tests?
All of the currently configured workflows passed.

Are there any other tests you are talking about?

**Pull request merged**